Packs/Venafi/Integrations/VenafiV2/VenafiV2.py

```diff
@@ -222,24 +217,23 @@ def get_certificates_command(client: Client, args: dict[str, Any]) -> CommandRes
         A CommandResult object with an outputs, raw_response and readable table, in case of a successful action.
     """
 
-    outputs: dict[str, Any] = {}
-    response = client._get_certificates(args)
-    if response:
-        outputs = delete_links_from_response(response)
-
-    human_readable = []
-    certificates = outputs.get('Certificates', [])
+    response = client.get_certificates(args)
+    certificates = response.get('Certificates', [])
+    readable_certificates = []
     for certificate in certificates:
-        certificate_details = get_human_readable_object(certificate)
-        human_readable.append(certificate_details)
+        readable_certificate_details = certificate.copy()
```

Comment:
As I saw implemented in v1, the id was inserted into the context without `{}`. Let's do it also in here, not only for the readable output.